### DISCUSSION TITLE
Why is this cosine scaling term in the SchNet implementation?

### DISCUSSION THREAD

**Discussion by NiklasKappel** [pyg-team/pytorch_geometric#7933] [Q&A]
[This line](https://github.com/pyg-team/pytorch_geometric/blob/master/torch_geometric/nn/models/schnet.py#L417) applies a cosine-based smooth cutoff to the filter values learned by the SchNet model. I guess it makes sense to have, as it reduces the interaction strength between more distant nodes. However, I can not find a reference to it in the [paper](https://arxiv.org/pdf/1706.08566.pdf) on which the implementation is based, so I wonder why it is there?

**Comment by rusty1s** [comment]
This is taken from here: https://github.com/atomistic-machine-learning/schnetpack/blob/2ed8d1a3b773f4ed2dbd50623d43d578ff0146f6/src/schnetpack/nn/cutoff.py#L14
